### ISSUE TITLE
Behaviour of shift_x shift_y in morphological base operations

### ISSUE THREAD

**Issue by pratapvardhan** [scikit-image/scikit-image#1309]
```python
import numpy as np
from skimage import morphology

img = np.ones((50, 50), dtype=np.uint8)
img[20:25,20:25]=0
img_d = morphology.dilation(img, morphology.selem.square(3), shift_x=2, shift_y=2)
```
![ski_shift](https://cloud.githubusercontent.com/assets/3757165/5471166/573baf16-8618-11e4-9b35-1548fcf439e5.PNG)

Is the actual image titled `Dilation` the intended behavior? I was thinking `Expected` image seems to be more appropriate..

**Comment by jni** [comment]
@pratapvardhan This behaviour is fixed by #1310, with the exception that `shift_` does nothing in the case of odd-sized elements.

If you have a good use case for eccentric elements, please open a PR for your translate function once #1310 is merged.

**Comment by pratapvardhan** [comment]
Thanks @jni
